enterprise/server/index.md

```diff
@@ -109,16 +115,19 @@ For other versions of Tomcat on Windows, check the Tomcat documentation for that
 
 If you're following the instructions for [Starting Jetty](https://www.eclipse.org/jetty/documentation/current/startup.html) for Jetty 9.4.5, the path to the configuration file can simply be supplied as a command option:
 
-    java -D"ephox.config.file=C:\config\file\location\application.conf" -jar C:\jetty\install\directory\start.jar
-
+```
+java -D"ephox.config.file=C:\config\file\location\application.conf" -jar C:\jetty\install\directory\start.jar
```

Comment:
should there be a space after -D ?

Reply:
Nope.

Reply:
See https://docs.oracle.com/en/java/javase/13/docs/specs/man/java.html#standard-options-for-java